### PR TITLE
Scroll chat to end of buffer when posting a new message

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

**Comment by swoolcock** [comment]
Worth adding a visual test?

File: osu.Game/Overlays/ChatOverlay.cs [hunk 1]
```diff
@@ -475,6 +475,8 @@ private void postMessage(TextBox textbox, bool newText)
 
             if (target == null) return;
 
+            currentChannelContainer.Child.ScrollToEnd();
```
Comment (smoogipoo) [inline osu.Game/Overlays/ChatOverlay.cs]: Is there any reason why this isn't done internal to DrawableChannel? Maybe via `NewMessagesArrived`?

Comment (peppy) [reply osu.Game/Overlays/ChatOverlay.cs]: Ah yes, this could be a better solution. I think we can tell using type-checking whether it's a local message.

**Comment by smoogipoo** [comment]
Doesn't work for info message/error message

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**